components/src/core/DropdownToolbar/DropdownToolbar.tsx

```diff
@@ -0,0 +1,181 @@
+                    primary={
+                        <Typography variant="h6" className={clsx(defaultClasses.title, classes?.title)}>
+                            {title}
+                        </Typography>
+                    }
+                    secondary={
+                        <Typography
+                            id={'dropdown-subtitle'}
+                            data-testid={'dropdown-subtitle'}
+                            ref={anchor}
+                            aria-haspopup="true"
+                            className={clsx(defaultClasses.subtitle, classes?.subtitle)}
+                            component={'div'}
+                            onClick={(): void => {
+                                setAnchorEl(anchor.current);
+                            }}
+                        >
+                            {subtitleLabel || ''}
+                            {menuItems && menuItems.length > 0 && (
+                                <ArrowDropDown className={clsx(defaultClasses.arrowDropdown, classes?.arrowDropdown)} />
+                            )}
+                            {customMenu && (
+                                <ArrowDropDown className={clsx(defaultClasses.arrowDropdown, classes?.arrowDropdown)} />
+                            )}
+                        </Typography>
+                    }
+                />
+                {props.children}
+            </Toolbar>
+
+            {menuItems && Boolean(anchorEl) && (
+                <Menu
+                    id="dropdown-toolbar-menu"
+                    data-testid="dropdown-toolbar-menu"
+                    elevation={0}
+                    getContentAnchorEl={null}
+                    anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
+                    transformOrigin={{ vertical: 'top', horizontal: 'center' }}
+                    anchorEl={anchorEl}
+                    open={Boolean(anchorEl)}
+                    onClose={(): void => setAnchorEl(null)}
+                    MenuListProps={{ style: { padding: 0 } }}
+                    PaperProps={{
+                        style: {
+                            maxHeight: MENU_ITEM_HEIGHT * 4.5,
+                            width: 200,
+                        },
+                    }}
+                >
+                    {!customMenu &&
+                        menuItems.map((item: ToolbarMenuItem, index: number) => (
+                            <MenuItem
+                                key={`Toolbar-Option-${index}`}
+                                id={`menu-item-${index}`}
+                                data-testid={`menu-item-${index}`}
+                                onClick={(): void => {
+                                    setAnchorEl(null);
+                                    item.onClick();
+                                }}
+                                className={clsx(defaultClasses.menuItem, classes?.menuItem)}
```

Comment:
You can avoid doing all of these conditional checks (`classes?.menuItem`) if you set a default value when you destructure:
```
const {classes = {} , ...} = props;
```